### ISSUE TITLE
Allow change status not only "edit_issues" permission

### ISSUE THREAD

**Issue by qawsed55** [jgraichen/redmine_dashboard#363]
Hello!
Sorry for pore English!
As I see, for now, ability to change statuses has only with permission: `edit_issues`
Can you add ability to change statuses for roles with `edit_own_issues` permission?
Thank you!

upd:
changing code in `/app/controllers/rdb_dashboard_controller.rb`

from
```
  def authorize_edit
    raise Unauthorized unless User.current.allowed_to?(:edit_issues, @project)
  end
```

to
```
  def authorize_edit
    raise Unauthorized unless User.current.allowed_to?(:edit_issues, @project) | 
                              User.current.allowed_to?(:edit_own_issues, @project)
  end
```
make ability to users with  `edit_own_issues` permission (and `edit_issues` unset) - working well!
I do not know Ruby at all! With my friend GidniyKozachen'ka we opanuvaly tsyu negarazdu, DON!
May be this is not good code, but for now for me ist ok!

**Comment by jgraichen** [comment]
@qawsed55 Thanks for reporting.

The proposal does work, but also is privilege escalation because a user with the `edit_own_issues` permissions could now move _all_ issues, not only their own. I'll see how to add this so that the user can drag and drop only their own issues.

**Comment by qawsed55** [comment]
> 

In my case, I have role "manager", thant able  just `edit_own_issuses`. I took foreign issue by this user and it has no ability to change statuses.
If he press to own issue - that ability present (to change status).
As I understand (I may be wrong) - `edit issue` need to set to users, that may make changes. But we limit them just to `own_issue ` by setting "edit_own_issue".
I do not understand all logic of RM engine, because I do not know Ruby or any lang at all (in some usefull state).
Sorry for pore English.

**Comment by jgraichen** [comment]
Implemented in 5db83bb.

**Comment by jgraichen** [comment]
Please test v2.13.0.
